docker/gcp_pudl_etl.sh

```diff
@@ -19,6 +26,18 @@ function authenticate_gcp() {
 }
 
 function run_pudl_etl() {
+    if [ -f commit_hash.txt ] && [ -f sha256_hash.txt ]; then
```

Comment:
This script runs inside the Docker container that produces the PUDL DB, so it doesn't make sense for it to be checking whether there's a cached copy to download.